bc_obps/registration/fixtures/mock/facility.json

```diff
@@ -319,22 +319,7 @@
       "longitude_of_largest_emissions": -123.5
     }
   },
-  {
-    "model": "registration.facility",
-    "pk": "459b80f9-b5f3-48aa-9727-90c30eaf3a59",
-    "fields": {
-      "operation": "3b5b95ea-2a1a-450d-8e2e-2e15feed96c9",
```

Comment:
Deleting because this operation is supposed to be an SFO and this is the second facility assigned to it

Reply:
you'll have made Dylan v happy with this - he was just complaining about it.